src/clients/Device.js

```diff
@@ -123,15 +123,15 @@ class Device extends EventEmitter {
 
     this._socket.on(
       'error',
-      (error: Error): void => this.disconnect(`socket error ${error.message}`),
+      (error: Error): void => this.disconnect(`socket error: ${error.message}`),
     );
     this._socket.on(
       'close',
-      (error: Error): void => this.disconnect(`socket close ${error.message}`),
+      (): void => this.disconnect('socket close'),
```

Comment:
'close' event have `had_error` : Boolean, arg. but no needs for this, because if there is an error, 'error' event will be dispatched before.